src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx

```diff
@@ -0,0 +1,53 @@
+import { i18n } from '@lingui/core';
+import { Box, Grid, makeStyles, Theme, Typography } from '@material-ui/core';
+import { CSSProperties } from '@material-ui/core/styles/withStyles';
+import React from 'react';
+import { Evaluation } from 'src/global-types';
+import { NON_BREAKING_SPACE } from 'src/shared/constants';
+import { EvaluationOutput } from 'src/shared/evaluation-output';
+import { MultilineOutput } from 'src/shared/multiline-output';
+import { FCProps } from 'src/shared/types/FCProps';
+import { Styles } from 'src/shared/types/Styles';
+
+interface OwnProps {
+  evaluation: '%future added value' | Evaluation | null;
+  evidence: string | null;
+}
+
+type Props = FCProps<OwnProps> & StyleProps;
+
+export function ProjectPeerReviewOutput({ evaluation, evidence, ...otherProps }: Props) {
```

Comment:
```suggestion
export function ProjectPeerReviewOutput(props: Props) {
const {evaluation, evidence} = props;
```

Reply:
Will open another PR to fix these thanks